tests/Unit/SubPageList/Lister/SubPageListTest.php

```diff
@@ -73,6 +73,7 @@ private function getRenderedList( SubPageList $subPageList, $titleText ) {
 				'showpage' => new ProcessedParam( 'showpage', true, false ),
 				'default' => new ProcessedParam( 'default', '', true ),
 				'limit' => new ProcessedParam( 'limit', 200, true ),
+				'sort' => new ProcessedParam( 'sort', 'asc', true ),
```

Comment:
Not entirely sure why this was added. I don't see why the test would fail without it just by looking at this PR, but then perhaps I'm being blind.

What is definitely the case is that the new behavior is not tested